### ISSUE TITLE
"Carousel" error: Exception Ticking [some animal on the map here]: System.InvalidOperationException: Sequence contains no elements

### ISSUE THREAD

**Issue by madmax3004** [cseelhoff/RimThreaded#470]
**Describe the bug**
The console throws an error for most if not all of the animals on the map, one by one (not instantly all of them, it takes a couple of seconds to go through them). This happens whenever I load up my save, and doesn't happen after that during the session.

**Steps to reproduce the behavior (VERY IMPORTANT)**
1: Load up my save.
2: See errors pop up.

**Error Log**
https://gist.github.com/HugsLibRecordKeeper/a6511821988b24660ac1db21f4c34deb

*EDIT:*
Errors in question start from here:
https://gist.github.com/HugsLibRecordKeeper/a6511821988b24660ac1db21f4c34deb#file-output_log-txt-L4727

**Save file**

[savefile.zip](https://github.com/cseelhoff/RimThreaded/files/6023873/savefile.zip)

**Mod list (Preferably a RimPy compatible list.)**

[modlist.zip](https://github.com/cseelhoff/RimThreaded/files/6023898/modlist.zip)

**Comment by Eldulac** [comment]
**[Tested on RT 1.3.2]**

I tested this issue on the save file posted here, and consulted PacMan and ZerosAesyr about the errors in an effort to find the cause.
It was determined that the likely cause for at least half of the errors may have been Android tiers, which is known to cause similar errors. The other half of the errors we were not able to conclude on.

When testing the save file, I did not experience any of the described errors, and I did not encounter any other errors either. The mod list was quite unstable though, and contained multiple mods that are incompatible with RT. This caused the game to be really slow.

It is worth noting that I received notifications for multiple mod updates when loading the save, which may have been the reason for the issue's disappearance.

**Conclusion**
The issue seems to have been fixed by mod updates, as I did not experience it while testing the attached save file. Mod incompatibilities and messy load order were the likely cause for the issue in the first place.
